src/app/utils/__tests__/limit-rate.spec.ts

```diff
@@ -35,7 +34,7 @@ describe('limitRate', () => {
     const handler = MockRateLimit.mock.calls[0][0]!.handler!
     const mockNext = jest.fn()
     const mockRes = expressHandler.mockResponse()
-    handler(expressHandler.mockRequest(), mockRes, mockNext)
+    handler(expressHandler.mockRequest(), mockRes, mockNext, {})
```

Comment:
Why is this needed?

Reply:
Haha, initially I just put that to stop the types from complaining, but since you commented...

`RateLimit` is a function which returns an `express` rate-limiting middleware. `RateLimit` accepts many options to configure this middleware, one of which is a custom `handler` to call when the rate limit is exceeded.

This particular test in `limit-rate.spec.ts` checks that our custom handler behaves correctly.

`express-rate-limit` passes 4 args to the custom `handler`: `req`, `res`, `next` as well as the object of `options` which was used to set up the middleware.

Previously, this test didn't pass the `options` to the handler when testing its behaviour. However, ideally the test should match the behaviour of `express-rate-limit` which is to pass those options. So I've updated the test to do this in [652077e](https://github.com/opengovsg/FormSG/pull/5568/commits/652077ecbf625d4e3ff729c6fe2a711dcd3f8ea8).